rpm/sailjail-permissions.spec

```diff
@@ -1,6 +1,6 @@
 Name:     sailjail-permissions
 Summary:  Permissions definitions for Sailjail
-Version:  1.0.0
+Version:  1.0.1
```

Comment:
If you touch this, it should be the roughly the new to be tagged version, i.e. 1.1.10.